Core/Pixel.swift

```diff
@@ -151,14 +154,43 @@ public class Pixel {
                             withAdditionalParameters params: [String: String?] = [:],
                             withHeaders headers: HTTPHeaders = APIHeaders().defaultHeaders,
                             onComplete: @escaping (Error?) -> Void = {_ in }) {
+        
+        var newParams = params
+        if isDebugBuild {
+            newParams["test"] = "1"
+        }
+        
         let formFactor = deviceType == .pad ? Constants.tablet : Constants.phone
         let url = appUrls
             .pixelUrl(forPixelNamed: pixel.rawValue, formFactor: formFactor)
-            .addParams(params)
+            .addParams(newParams)
         
         Alamofire.request(url, headers: headers).validate(statusCode: 200..<300).response { response in
             Logger.log(items: "Pixel fired \(pixel.rawValue)")
             onComplete(response.error)
         }
     }
+    
+    /// Don't load a pixel unless you intend to fire it!
+    public static func load(pixel: PixelName) -> TimedPixel {
+        return TimedPixel(pixel)
+    }
```

Comment:
Looks good @brindy! 

I'd only consider either removing this method (and use `TimedPixel` directly) or tweaking the name to something like `makeTimedPixel` (in case of the latter, I assume we can't make `TimedPixel`'s `init` method fileprivate since it is used in the tests).